doc/UserGuide.md

```diff
@@ -45,8 +45,8 @@ Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
 > Persons can have any number of tags (including 0)
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe pp/1234567 e/betsycrowe@gmail.com pa/Newgate Prison t/criminal t/friend`
+* `add John Doe p/98765432 e/johnd@gmail.com a/123, John Street, #01-01, 123456`
+* `add Betsy Crowe pp/1234567 e/betsycrowe@gmail.com pa/123, Newgate Prison, #12-12, 123342 t/criminal t/friend`
 
```

Comment:
Great! You changed the user guide... but what about the in-program help? Such as the add command usage?